From 697f8f61e8e99594172b0bae4f7e397836218021 Mon Sep 17 00:00:00 2001
From: Alde Rojas <hello@alde.dev>
Date: Wed, 27 Sep 2023 19:42:32 -0500
Subject: [PATCH] Set max framerate for offline clients

---
 Engine/Src/UnGame.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Engine/Src/UnGame.cpp b/Engine/Src/UnGame.cpp
index ae74f08..e42f016 100644
--- a/Engine/Src/UnGame.cpp
+++ b/Engine/Src/UnGame.cpp
@@ -3322,6 +3322,12 @@ FLOAT UGameEngine::GetMaxTickRate()
 		FLOAT MaxClientFrameRate = (GLevel->NetDriver->ServerConnection->CurrentNetSpeed > 10000) ? GLevel->GetLevelInfo()->MaxClientFrameRate : 90.f;
 		return Clamp((GLevel->NetDriver->ServerConnection->CurrentNetSpeed - GLevel->NetDriver->ServerConnection->CurrentVoiceBandwidth)/GLevel->GetLevelInfo()->MoveRepSize, 10.f, MaxClientFrameRate);
 	}
+	else
+	if( GLevel )
+	{
+		// We're an offline client, use MaxClientFrameRate instead of being unbounded.
+		return Max(10.0f, GLevel->GetLevelInfo()->MaxClientFrameRate);
+	}
 	else
 		return 0;
 	unguard;
-- 
2.31.1.windows.1

