From aaf3f01d658a25ea7611006dc436bbda3ec2cda5 Mon Sep 17 00:00:00 2001
From: Alde Rojas <hello@alde.dev>
Date: Fri, 6 Oct 2023 21:27:56 -0500
Subject: [PATCH] Increase refresh rate for OpenGL up to 600

---
 OpenGLDrv/Src/OpenGLRenderDevice.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OpenGLDrv/Src/OpenGLRenderDevice.cpp b/OpenGLDrv/Src/OpenGLRenderDevice.cpp
index d87367f..ee87ea8 100644
--- a/OpenGLDrv/Src/OpenGLRenderDevice.cpp
+++ b/OpenGLDrv/Src/OpenGLRenderDevice.cpp
@@ -485,7 +485,7 @@ UBOOL UOpenGLRenderDevice::SetRes(UViewport* Viewport,INT NewX,INT NewY,UBOOL Fu
 			}
 		}
 
-		DWORD RefreshRate = DesiredRefreshRate ? Clamp<DWORD>( DesiredRefreshRate, 60, 100 ) : 0;
+		DWORD RefreshRate = DesiredRefreshRate ? Clamp<DWORD>( DesiredRefreshRate, 60, 600 ) : 0;
 
 #if UNICODE
 		if( !GUnicodeOS )
-- 
2.31.1.windows.1

